doc/classes/Input.xml

```diff
@@ -295,6 +295,15 @@
 				[/codeblocks]
 			</description>
 		</method>
+		<method name="register_axis_as_button">
+			<return type="void" />
+			<param index="0" name="device" type="int" />
+			<param index="1" name="axis" type="int" enum="JoyAxis" />
+			<param index="2" name="trigger_value" type="float" />
```

Comment:
"trigger value" is usually called "deadzone", unless it's something different in this case?

Reply:
Thanks, I have renamed it.